spec/requests/admin/admin_portals_spec.rb

```diff
@@ -22,4 +22,22 @@
       expect(response.body).not_to include("Config: Profile Setup")
     end
   end
+
+  describe "Last deployed and Lastest Commit ID card" do
+    it "shows 'Not Available' if the Last deployed time is missing" do
+      ENV["HEROKU_RELEASE_CREATED_AT"] = ""
```

Comment:
non blocking: this sets the variable globally for any test

Reply:
@rhymes I tried to set it so many other like using         
`allow(ENV).to receive(:[]).with("HEROKU_RELEASE_CREATED_AT").and_return("")` but I just couldn't get it to work. I'll look into it again tomorrow.

Reply:
Ah got it! No worries, we can keep it like this as well :)

Reply:
@Ridhwana Maybe something like this?

```ruby
stub_const("ENV", ENV.to_h.merge("HEROKU_RELEASE_CREATED_AT" => ""))
```

Reply:
Thanks @citizen428 this worked!